crates/pessimistic-proof/src/local_state.rs

```diff
@@ -197,6 +200,10 @@ impl LocalNetworkState {
         // TODO: implement batch `verify_and_update` for the LBT
         for (token, (old_balance, balance_path)) in &multi_batch_header.balances_proofs {
             let new_balance = new_balances[token];
+            if new_balance > U512::from(U256::MAX) {
+                return Err(ProofError::BalanceOverflowInBridgeExit);
+            }
+            let new_balance = U256::from(new_balance);
```

Comment:
Nitpick: is there something `U256::try_from` to roll the check and conversion into one call? I see `U256::uint_try_from` exists but not sure how that works.